account_vendor_bank_account_default/README.rst

```diff
@@ -0,0 +1,120 @@
+   !! source digest: sha256:446b86643c42066a7533bf64a0cf696042ce0e2d902183ec50d5f9d9c3e255ed
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fbank--payment-lightgray.png?logo=github
+    :target: https://github.com/OCA/bank-payment/tree/17.0/account_vendor_bank_account_default
+    :alt: OCA/bank-payment
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/bank-payment-17-0/bank-payment-17-0-account_vendor_bank_account_default
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runboat-Try%20me-875A7B.png
+    :target: https://runboat.odoo-community.org/builds?repo=OCA/bank-payment&target_branch=17.0
+    :alt: Try me on Runboat
+
+|badge1| |badge2| |badge3| |badge4| |badge5|
+
+This module allows you to set a default bank account on partners for
+their vendor bills.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+-  Go to a partner form view, and edit the Default Bank Account in the
+   Sales and Purchase tab. The partner must be of type 'company', or an
+   individual without a parent company.
+
+   **Note:** If you do not set this value, it will be equal to the first
+   bank account created in the contact. This is the standard behaviour
+   of Odoo
+
+-  You can disable the default bank account option at contact level,
+   unmarking the "Has Default Bank Account" check.
+
+Usage
+=====
+
+To use this module, you need to:
+
+1. Create a vendor bill and select a partner with a default bank
+   account. Its recipient bank account will be the default one.
+
+Known issues / Roadmap
+======================
+
+-  This module depends on account_payment_partner. If the payment mode
+   of the invoice has a payment method that does not require a bank
+   account, the default bank account will be empty.
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/bank-payment/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us to smash it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/bank-payment/issues/new?body=module:%20account_vendor_bank_account_default%0Aversion:%2017.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
+=======
+
+Authors
+-------
+
+* Sygel
```

Comment:
Odoo Community Association (OCA) it's necessary?

Reply:
I think it's not necessary because it is autogenerated from the manifest, and there is mentioned OCA, so the readme generator must be ignoring it